src/ActionModal/components/DelegationModal.vue

```diff
@@ -64,12 +64,7 @@
         type="text"
         readonly
       />
-      <template
-        v-if="
-          network.type !== `polkadot` ||
-          (addressRole && addressRole !== `stash`)
-        "
-      >
+      <template v-if="session.addressRole !== `stash`">
```

Comment:
Good catch. Nevertheless, it makes more sense to place this check in the `TmFormGroup` above. Like here: https://github.com/luniehq/lunie/pull/4125/files#diff-19dd02dcdf9da85fced29cbd416710b4R56

Reply:
But yeah, this other check needs to be removed completely. Good eye